lib/mwoffliner.lib.js

```diff
@@ -25,6 +25,7 @@ var os = require( 'os' );
 var crypto = require( 'crypto' );
 var unicodeCutter = require( 'utf8-binary-cutter' );
 var htmlMinifier = require('html-minifier');
+const fetch = require('node-fetch')
```

Comment:
A bit sad, that we introduce here a new download framework/soltuion as we already have download primitives in the code.